lib/save.js

```diff
@@ -0,0 +1,18 @@
+var thenify = require('thenify')
+var fs = require('fs')
+var writeFile = thenify(fs.writeFile)
+
+module.exports = function save (pkg, installedPackages, saveType) {
+  var packageJson = pkg.pkg
+  packageJson[saveType] = packageJson[saveType] || {}
+  installedPackages.forEach(function (dependency) {
+    // Assumes package name of in the format of "package-name@x.x.x"
+    // TODO: Support other install sources (Github, scoped pkgs etc.)
```

Comment:
Yeah, you're correct. I'll pick this up tomorrow, I have to head out the door now.